pay-api/src/pay_api/services/fas/routing_slip_status_transition_service.py

```diff
@@ -55,7 +55,7 @@ class RoutingSlipStatusTransitionService:  # pylint: disable=too-many-instance-a
             RoutingSlipCustomStatus.CANCEL_WRITE_OFF_REQUEST.custom_status,  # pylint: disable=no-member
         ],
         RoutingSlipStatus.REFUND_AUTHORIZED.value: [],
-        RoutingSlipStatus.REFUND_COMPLETED.value: [],
+        RoutingSlipStatus.REFUND_PROCESSED.value: [],
```

Comment:
Have you handled the changes in FAS-UI for this?

Reply:
yes